src/com/trilead/ssh2/packets/PacketKexInit.java

```diff
@@ -20,10 +20,10 @@
 
 	KexParameters kp = new KexParameters();
 
-	public PacketKexInit(CryptoWishList cwl, SecureRandom rnd)
+	public PacketKexInit(CryptoWishList cwl)
```

Comment:
Any reason why the second argument has been removed to then create a new instance on every call?